projects/dxilconv/unittests/CMakeLists.txt

```diff
@@ -15,6 +15,7 @@ target_link_libraries(dxilconv-tests PRIVATE
   LLVMSupport
   LLVMOption
   ${TAEF_LIBRARIES}
+  shlwapi
```

Comment:
I'm curious what this is and how it enables the arm compile. I'm curious the best way to pronounce it without sounding drunk too, but that's less important 😝

Reply:
It is a library that is part of Windows OS. SHLWAPI stands for Shell Light-Weight Application Programming Interface so "shell lightweight api" is probably the safest bet as far as pronunciation goes :). I am not sure why is needs to be specified explicitly for arm build and not for other flavors, but I have seen similar things happen for ARM64 build in our other project.